.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: CI
+on:
+  push:
+  pull_request:
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.2', '7.3', '7.4']
```

```suggestion
        php: ['7.3', '7.4', '8.0']
```
I don't know if this already works with ubuntu-latest, but we should try to run the tests on PHP8.0 and drop 7.2 as it is already EoL